### ISSUE TITLE
Remove fake status bar for iOS

### ISSUE THREAD

**Issue by NiklasMerz** [apache/cordova-plugin-inappbrowser#604]
There are some issues with iPhones with a notch and the status bar. PR #546 tries to fix that by removing the status bar for these devices.

This is not a proper fix, it just hides the fake status bar for iPhone X and other devices with notches.

The proper fix should make the fake statusbar the size of the notch to be consistent, or just remove the fake statusbar for all devices.

_Originally posted by @jcesarmobile in https://github.com/apache/cordova-plugin-inappbrowser/pull/546#issuecomment-542434369_

Discussion in #546 tends to the removal of the status bar in major release.

**Comment by NiklasMerz** [comment]
We should try to get a consistent behaviour with Android. See #598

**Comment by ashtt** [comment]
Around what time/month or in which version should we expect this fix ?

**Comment by jcesarmobile** [comment]
When somebody sends a pull requests that fixes it and it gets merged and a release process is started and finish successfully.
Version >= 4.0.0

**Comment by benvallack** [comment]
So how do we use this version? Also if we use _self to get the pages to load in the cordova view itself, would that improve things? Trouble is I see this issue when I try that: https://stackoverflow.com/questions/58321114/why-i-get-the-console-warning-process-kill-returned-unexpected-error-1-when

**Comment by NiklasMerz** [comment]
This has nothing to do with this issue. A version which removes the statusbar does not exist, yet.

**Comment by sardapv** [comment]
I have fixed that notch issue for iphoneX+ series (https://issues.apache.org/jira/browse/CB-13956)
It handles both landscape and portrait orientation change as well (right left area both). Inappbrowser scales first checking all other factors. Only problem I found is when I change to landscape mode I see status bar pane but no battery wifi or any symbol in it. Not a major issue I suppose. Currently those changes are on my local.

@jcesarmobile   @NiklasMerz 
It does similar thing that is in PR https://github.com/apache/cordova-plugin-inappbrowser/pull/546 
But does it better with orientation change even afterIAB is opened.
